Patch to fix SBAT format.
Already snet upstream and accepted there.

commit 50348e1104a97db27253ddd1d5e8522fa7aa63bc
Author: Steve McIntyre <93sam@debian.org>
Date:   Fri Mar 26 15:06:08 2021 +0000

    Tweak the SBAT output for a vendor string
    
    The format is meant to be "<project>.<vendor>" with a period as a
    separator.
    
    Signed-off-by: Steve McIntyre <93sam@debian.org>

diff --git a/plugins/uefi-capsule/efi/generate_sbat.py b/plugins/uefi-capsule/efi/generate_sbat.py
index 18f7c4c7..1f12ca11 100755
--- a/plugins/uefi-capsule/efi/generate_sbat.py
+++ b/plugins/uefi-capsule/efi/generate_sbat.py
@@ -51,7 +51,7 @@ def _generate_sbat(args):
 
         # distro specifics, falling back to the project defaults
         sfd.write(
-            "{0}-{1},{2},{3},{4},{5},{6}\n".format(
+            "{0}.{1},{2},{3},{4},{5},{6}\n".format(
                 args.project_name,
                 args.sbat_distro_id,
                 args.sbat_distro_generation or args.sbat_generation,
